release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -295,6 +296,25 @@ For more information, see xref:../operators/operator_sdk/osdk-token-auth.adoc#os
 [id="ocp-4-14-developer-perspective"]
 ==== Developer Perspective
 
+With this release, there are several updates to the *Developer* perspective of the web console. You can now perform the following actions:
+
+* Change the default timeout period for the web terminal for your current session. For more information, see xref:../web_console/web_terminal/configuring-web-terminal.adoc#odc-configure-web-terminal-timeout-session_configuring-web-terminal[Configuring the web terminal timeout for a session].
+* Test {serverlessproductshortname} functions in the web console from the *Topology* view and the Serverless Service *List* and *Detail* pages so you can use a {serverlessproductshortname} function with a CloudEvent or HTTP request.
+* View status, start time, and duration of the latest build for `BuildConfigs` and Shipwright builds. You can also view this information on the *Details* page.
```

Comment:
```suggestion
* View status, start time, and duration of the latest build for `BuildConfigs` and Shipwright builds. You can also view this information about the *Details* page.
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#i

Reply:
I think in this case `on` is okay because you view the information on that page.